xarray/plot/utils.py

```diff
@@ -51,8 +51,8 @@
 ROBUST_PERCENTILE = 2.0
 
 # copied from seaborn
-_MARKERSIZE_RANGE = (18.0, 72.0)
-_LINEWIDTH_RANGE = (1.5, 6.0)
+_MARKERSIZE_RANGE = (18.0, 36.0, 72.0)
```

Comment:
It's not really a range anymore, but since it is internal only I don't care much.